stdlib/public/core/Runtime.swift.gyb

```diff
@@ -72,10 +70,20 @@ func _stdlib_atomicCompareExchangeStrongPtr<T>(
   object target: UnsafeMutablePointer<UnsafeMutablePointer<T>${optional}>,
   expected: UnsafeMutablePointer<UnsafeMutablePointer<T>${optional}>,
   desired: UnsafeMutablePointer<T>${optional}) -> Bool {
-  return _stdlib_atomicCompareExchangeStrongPtr(
-    object: unsafeBitCast(target, to: _PointerToPointer.self),
-    expected: unsafeBitCast(expected, to: _PointerToPointer.self),
-    desired: unsafeBitCast(desired, to: Optional<UnsafeRawPointer>.self))
+  return target.withMemoryRebound(
+    to: Optional<UnsafeRawPointer>.self,
+    capacity: 1
+  ) { target in
+    return expected.withMemoryRebound(
+      to: Optional<UnsafeRawPointer>.self,
+      capacity: 1
+    ) { expected in
+      return _stdlib_atomicCompareExchangeStrongPtr(
+        object: target,
+        expected: expected,
+        desired: UnsafeRawPointer(desired))
```

Comment:
Purely style. I don't like relying on pointer-to-pointer conversions in pure Swift code.